packages/patrol/android/src/main/kotlin/pl/leancode/patrol/PatrolJUnitRunner.java

```diff
@@ -1,3 +1,7 @@
+// Terminology note:
+// "Run a test" is used interchangeably with "execute a test".
+// "Run a Dart test" is used interchangeably with "request execution of a Dart test" and "execute Dart test".
```

Comment:
Maybe instead of using interchangeable terms stick to one?

Reply:
I was thinking about this, but it'd make the docs not nice to read.

Also, and it may not be the greatest argument, [but Flutter does it](https://github.com/flutter/flutter/blob/76a9f683259ea4ad1844250c9772b5437e04c200/packages/flutter/lib/src/foundation/change_notifier.dart#L37-L38) :D

Reply:
Generally I am not a fan of using many terms when referring to the same thing. It is just a source of confusion.